cluster-autoscaler/cloudprovider/kubemark/kubemark_other.go

```diff
@@ -1,3 +1,5 @@
+// +build ignore
```

Comment:
shouldn't that be +build !linux ? Also I'm pretty sure it needs to go below boilerplate